packages/flutter_tools/test/commands/analyze_continuously_test.dart

```diff
@@ -69,44 +69,6 @@ void main() {
   }, overrides: <Type, Generator>{
     OperatingSystemUtils: () => os
   });
-
-  testUsingContext('--preview-dart-2', () async {
-    const String contents = "StringBuffer bar = StringBuffer('baz');";
-    tempDir.childFile('main.dart').writeAsStringSync(contents);
-    server = new AnalysisServer(dartSdkPath, <String>[tempDir.path], previewDart2: true);
-
-    int errorCount = 0;
-    final Future<bool> onDone = server.onAnalyzing.where((bool analyzing) => analyzing == false).first;
-    server.onErrors.listen((FileAnalysisErrors errors) {
-      errorCount += errors.errors.length;
-    });
-
-    await server.start();
-    await onDone;
-
-    expect(errorCount, 0);
-  }, overrides: <Type, Generator>{
-    OperatingSystemUtils: () => os
-  });
-
```

Comment:
why are you deleting the dart 2 mode for this test also?

Reply:
Good catch - I was a bit aggressive w/ deletes here. I restore one test in this file, and one in the file below.